dashboard/src/t5gweb/cache.py

```diff
@@ -61,7 +61,7 @@ def get_cases(cfg):
 def get_escalations(cfg):
     '''Get cases that have been escalated from Smartsheet'''
     cases = libtelco5g.redis_get('cases')
-    if cases is None:
+    if cases is None or cfg['smartsheet_access_token'] is not None or cfg['smartsheet_access_token'] != '':
```

Comment:
Would we want the last two clauses to be inversed? Ex: `cfg['smartsheet_access_token] is None` so if the variable isn't defined the redis variable is set to None and the function returns